packages/cli/lib/oauth.js

```diff
@@ -31,7 +31,9 @@ const handleServerOnProcessEnd = server => {
 };
 
 const authorize = async oauthManager => {
-  open(buildAuthUrl(oauthManager), { url: true });
+  if (process.env.BROWSER !== 'none') {
+    open(buildAuthUrl(oauthManager), { url: true });
+  }
 
```

Comment:
Not entireley sure if it makes sense to, but I'm wondering if we should log the output here as we do with PAKs